README.rst

```diff
@@ -1,63 +1,73 @@
-What is py-esp32-ulp?
----------------------
+=====================
+py-esp32-ulp
+=====================
 
-It is an assembler toolchain for the ESP32 ULP (Ultra Low-Power) Co-Processor,
-written in MicroPython.
+.. image:: ../../actions/workflows/run_tests.yaml/badge.svg
+   :height: 20px
+   :target: ../../actions/workflows/run_tests.yaml
+   :alt: Build Status
```

Comment:
how about moving this to the very top or bottom, so it can be easily cut away when using this for the pypi package description?

Reply:
Good idea. I moved them now and updated `setup.py` to strip them away.